### PR TITLE
Fix trade tariff repo names

### PR DESCRIPTION
**Pull request by MatMoore**
The repo is different for frontend/backend, so I've split up the
function that maps these into one for frontend and one for backend.

### CODE REVIEW

File: src/popup/external_links.js [hunk 1]
```diff
@@ -3,22 +3,34 @@ var Popup = Popup || {};
 // With the content item we can generate a bunch of external links.
 Popup.generateExternalLinks = function(contentItem, env) {
 
-  // Not all publishing_app/rendering_app name corresponds to the name of the
+  // Not all publishing_apps name corresponds to the name of the
   // alphagov repo.
-  function appNameToRepo(appName) {
+  function appNameToPublishingRepo(appName) {
```
Comment (tijmenb) [inline src/popup/external_links.js]: Maybe `appNameToPublishingRepo` -> `renderingAppNameToRepo`?

File: src/popup/external_links.js [hunk 2]
```diff
@@ -3,22 +3,34 @@ var Popup = Popup || {};
...
+      smartanswers: 'smart-answers',
+      tariff: 'trade-tariff-backend'
+    };
+
+    return APP_NAMES_TO_REPOS[appName] || appName;
+  }
+
+  // Not all rendering_apps name corresponds to the name of the
+  // alphagov repo.
+  function appNameToRenderingRepo(appName) {
```
Comment (tijmenb) [inline src/popup/external_links.js]: Same here, `renderingAppNameToRepo`?

**Comment by MatMoore** [comment]
The tests check a rendering/publishing app github link but it doesn't look like it exercises this mapping so I've added a couple of more.

**Comment by tijmenb** [comment]
Woohoo. Nice!

**Pull request merged**